helios-client/src/main/java/com/spotify/helios/common/descriptors/RolloutTask.java

```diff
@@ -17,18 +17,21 @@
 
 package com.spotify.helios.common.descriptors;
 
+import static com.google.common.base.Preconditions.checkNotNull;
+
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class RolloutTask extends Descriptor {
 
   public enum Action {
     UNDEPLOY_OLD_JOBS,
     DEPLOY_NEW_JOB,
     AWAIT_RUNNING,
+    FORCE_UNDEPLOY_JOBS,
```

Comment:
If you're actually curious I attempted to explain it in https://github.com/spotify/helios/pull/986/commits, and by answering @davidxia's question on this PR. Point taken regarding documenting it in code though.